lms/djangoapps/certificates/views/webview.py

```diff
@@ -524,10 +525,9 @@ def render_html_view(request, user_id, course_id):
         course_key = CourseKey.from_string(course_id)
         user = User.objects.get(id=user_id)
         course = modulestore().get_course(course_key)
-        national_id = NationalId.objects.get(user=user_id)
 
     # For any other expected exceptions, kick the user back to the "Invalid" screen
-    except (InvalidKeyError, ItemNotFoundError, User.DoesNotExist, NationalId.DoesNotExist) as exception:
+    except (InvalidKeyError, ItemNotFoundError, User.DoesNotExist) as exception:
```

Comment:
@pepeportela Don't remove it from here. We still need to check it works in the `try` block.